user_guide.adoc

```diff
@@ -3569,34 +3570,34 @@ types:
         if: has_right_child != 0
 ----
 
-We've changed only three lines. We've enforced parent of the node in
+We've changed only three lines. We've enforced the parent of the node in the
 `left_child` and `right_child` attributes to be passed as `_parent`,
-not `this`. This, effectively, continues passing reference to original
-node's parent, which is a `tree` type object, over and over the whole
-recursive structure. This way one can access structure's root by just
+not `this`. This, effectively, continues passing a reference to the original
+node's parent, which is a `tree` type object, over and over in the whole
+recursive structure. This way one can access the structure's root by just
 using `_parent`. Naturally, we've done exactly that to get ourselves
 `chunk_size` by just using `size: _parent.chunk_size`.
 
 ==== Omitting parent
 
-In some cases, you'd rather want some object to don't have any parent
-at all. Primary use case for that is to make sure that some
-instantiation it does not affect parent type. In many cases, resorting
+In some cases, you want some object to not have any parent
```

Comment:
Shouldn't it be

```suggestion
In some cases, you want some object not to have any parent
```

?

Reply:
Thanks. I've gone for your second suggestion.